scipy/stats/tests/test_stats.py

```diff
@@ -3271,6 +3274,34 @@ def test_empty_1d(self):
             stats.kurtosis([])
 
 
+@hypothesis.strategies.composite
```

Comment:
To commit before merge.
```suggestion
@pytest.mark.xslow
@hypothesis.strategies.composite
```

Reply:
This does not seem needed, because this isn't a test function. And the `test_pvalue_ci` it's used from already has the `xslow` marker.

Reply:
Oops!